runtime/dom-context.js

```diff
@@ -208,7 +208,12 @@ class SetDomContext {
     Object.keys(this._contextBySubId).forEach(subId => {
       let templateForSubId = template[subId] || template[''];
       if (templateForSubId) {
-        this._contextBySubId[subId].stampTemplate(templateForSubId, eventHandler, eventMapper);
+        // TODO: Clean up this hack that allows the subId to reach the event handler.
```

Comment:
Don't want to hack into event handling or dom-context in general unless we have a really good reason. Currently, event handlers forward `key` and `data` properties from elements. I believe you can put the `subId` into the `key` property of the cells, which is the intended usage. Sorry if I misunderstood something that makes that unfeasible.

Reply:
Ahh I see, `key` is probably the same as what I was asking about here: https://github.com/PolymerLabs/arcs/issues/1022

Reply:
Seems to be exactly that 😛